packages/launcher/src/widget.tsx

```diff
@@ -298,7 +301,11 @@ function Card(
       <div className="jp-LauncherCard-icon">
         {kernel ? (
           item.kernelIconUrl ? (
-            <img src={item.kernelIconUrl} className="jp-Launcher-kernelIcon" />
+            <img
+              src={item.kernelIconUrl}
+              className="jp-Launcher-kernelIcon"
+              alt={trans.__('kernel icon image')}
```

Comment:
Same reason for captialization, and I recommend matching the mention of `icon` throughout. Lots of assistive tech usually announces that it is an image on its own.

```suggestion
              alt={trans.__('Kernel icon')}
```